examples/demo-app/src/app.js

```diff
@@ -48,7 +48,8 @@ const KeplerGl = require('kepler.gl/components').injectComponents([
 import sampleTripData from './data/sample-trip-data';
 import sampleGeojson from './data/sample-small-geojson';
 import sampleGeojsonPoints from './data/sample-geojson-points';
-import sampleH3Data from './data/sample-hex-id-csv';
+import sampleGeojsonConfig from './data/sample-geojson-config';
```

Comment:
this file doesn't exist or your forgot to push it to the repo. That's why the netlify build is failing